.stylelintrc.js

```diff
@@ -1,7 +1,12 @@
+// Ensure we use all available babel parser plugins, but not the type annotation
+// ones since stylelint handles that itself.
+const { buildOptions } = require('@codemod/parser')
+const parserPlugins = buildOptions().plugins.filter(plugin => plugin !== 'typescript' && plugin !== 'flow')
```

Comment:
This was needed because `nullishCoalescingOperator` isn't enabled by default in stylelint yet.